cts/cts-exec.in

```diff
@@ -510,150 +510,150 @@ class ExecTests(Tests):
-            test.add_cmd(args=f'-c cancel -r test_rsc -a monitor -i 1s{self._action_timeout}'
+            test.add_cmd(args=f'-c cancel -r test_rsc -a monitor -i 1s{self._action_timeout} '
                          '-l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:monitor rc:not running op_status:Cancelled" ')
             test.add_cmd(args=f'-l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:monitor rc:not running op_status:Done" {self._action_timeout}',
                          expected_exitcode=ExitStatus.TIMEOUT)
             test.add_cmd(args=f'-l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:monitor rc:ok op_status:Done" {self._action_timeout}',
                          expected_exitcode=ExitStatus.TIMEOUT)
-            test.add_cmd(args=f'-c unregister_rsc -r test_rsc {self._action_timeout}'
+            test.add_cmd(args=f'-c unregister_rsc -r test_rsc {self._action_timeout} '
                          '-l "NEW_EVENT event_type:unregister rsc_id:test_rsc action:none rc:ok op_status:Done" ')
 
         # Cancel non-existent operation on a resource
         test = self.new_test("cancel_non_existent_op", "Attempt to cancel the wrong monitor operation, verify expected failure")
-        test.add_cmd(args=f'-c register_rsc -r test_rsc -C ocf -P pacemaker -T Dummy {self._action_timeout}'
+        test.add_cmd(args=f'-c register_rsc -r test_rsc -C ocf -P pacemaker -T Dummy {self._action_timeout} '
                      '-l "NEW_EVENT event_type:register rsc_id:test_rsc action:none rc:ok op_status:Done" ')
-        test.add_cmd(args=f'-c exec -r test_rsc -a start {self._action_timeout}'
+        test.add_cmd(args=f'-c exec -r test_rsc -a start {self._action_timeout} '
                      '-l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:start rc:ok op_status:Done" ')
-        test.add_cmd(args=f'-c exec -r test_rsc -a start {self._action_timeout}'
+        test.add_cmd(args=f'-c exec -r test_rsc -a start {self._action_timeout} '
                      '-l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:start rc:ok op_status:Done" ')
-        test.add_cmd(args=f'-c exec -r test_rsc -a monitor -i 1s {self._action_timeout}'
+        test.add_cmd(args=f'-c exec -r test_rsc -a monitor -i 1s {self._action_timeout} '
                      '-l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:monitor rc:ok op_status:Done" ')
         test.add_cmd(args=f'-l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:monitor rc:ok op_status:Done" {self._action_timeout}')
         # interval is wrong, should fail
-        test.add_cmd(args=f'-c cancel -r test_rsc -a monitor -i 2s{self._action_timeout}'
-                     ' -l "NEW_EVENT event_type:exec_complete rsc_id:test_rsc action:monitor rc:not running op_status:Cancelled" ',
+        test.add_cmd(args=f'-c cancel -r test_rsc -a monitor -i 2s{self._action_timeout} '
```

Comment:
I'd rather the variable not have a space at the beginning of its value either. We end up with several lines like this. Some of them are obvious like this one, and others are concatenated literals (where `f"{self._action_timeout}"` occurs on the second line, after a first line that doesn't end with a space).

Reply:
It's not important and you can go ahead and merge this if you want. That's just my preference for cleanness.

Reply:
Yeah you're right, that's annoying too.  I'll remove those spaces, run the regression tests, and if that passes I'll merge it.